src/main/java/dao3/dao/jdbc/ManufacturerDaoJDBCImpl.java

```diff
@@ -0,0 +1,112 @@
+package dao3.dao.jdbc;
+
+import dao3.dao.ManufacturerDao;
+import dao3.exception.DataProcessingException;
+import dao3.lib.Dao;
+import dao3.model.Manufacturer;
+import dao3.util.ConnectionUtil;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import static dao3.db.Database.MANUFACTURERS_TAB_NAME;
+
+@Dao
+public class ManufacturerDaoJDBCImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query =
+                "INSERT INTO " + MANUFACTURERS_TAB_NAME + " (manufacturer_name, manufacturer_country) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement stmt = connection
```

Comment:
bad name of variable
```suggestion
            PreparedStatement statement = connection
```